web/routes.go

```diff
@@ -85,3 +105,24 @@ func jobStatusHandler(w http.ResponseWriter, r *http.Request) {
 	status := executer.GetTaskStatus(id)
 	w.Write([]byte(status.String()))
 }
+
+func formHandler(w http.ResponseWriter, r *http.Request) {
+	const tpl = `
```

Comment:
Please put this in an actual html file using whatever conventions Go has for static files.